### ISSUE TITLE
Does not compile because `structopt 0.2.17` has been yanked

### ISSUE THREAD

**Issue by KizzyCode** [r-darwish/topgrade#169]
I cannot not compile/install topgrade because `structopt 0.2.17` has been yanked

```
    Updating crates.io index
  Installing topgrade v2.1.0
error: failed to compile `topgrade v2.1.0`, intermediate artifacts can be found at `/tmp/cargo-installV9NXqo`

Caused by:
  failed to select a version for the requirement `structopt = "^0.2.17"`
  candidate versions found which didn't match: 0.2.16, 0.2.15, 0.2.14, ...
  location searched: crates.io index
required by package `topgrade v2.1.0`
```

**Comment by r-darwish** [comment]
I just published version 2.2.1

**Comment by KizzyCode** [comment]
Awesome; thanks 😊
